src/ng2-auto-complete.component.ts

```diff
@@ -138,10 +140,8 @@ export class Ng2AutoCompleteComponent implements OnInit {
   /**
    * constructor
    */
-  constructor(
```

Comment:
Let us not change this style, unless you do have a clear reason to do so. e.g., Industry standard or community recommendation.